docs/content/_index.md

```diff
@@ -69,28 +70,37 @@ Scaffolding a Boltzmann service in /Users/cj/code/hello
     updating dependencies...
         adding are-we-dev @ ^1.0.0
         adding uuid @ ^8.3.0
+        adding @hapi/iron @ ^6.0.0
         adding culture-ships @ ^1.0.0
-        adding find-my-way @ ^2.2.1
+        adding find-my-way @ ^3.0.4
         adding bole @ ^4.0.0
         adding ajv @ ^6.12.2
         adding dotenv @ ^8.2.0
         adding accepts @ ^1.3.7
         adding cookie @ ^0.4.1
         adding tap @ ^14.10.7 (dev)
-        adding @hapi/shot @ ^4.1.2 (dev)
+        adding ansi-escapes @ ^4.3.1 (dev)
+        adding supports-hyperlinks @ ^2.1.0 (dev)
+        adding get-function-location @ ^2.0.0 (dev)
+        adding @hapi/shot @ ^5.0.4 (dev)
         adding bistre @ ^1.0.1 (dev)
         adding eslint @ ^7.1.0 (dev)
         adding @typescript-eslint/parser @ ^3.1.0 (dev)
         adding @typescript-eslint/eslint-plugin @ ^3.1.0 (dev)
         adding eslint-config-prettier @ ^6.11.0 (dev)
         adding eslint-plugin-prettier @ ^3.1.3 (dev)
         adding prettier @ ^2.0.5 (dev)
+        adding nodemon @ ^2.0.4 (dev)
     writing updated package.json...
     running package install...
-Boltzmann at 0.1.2 with githubci, ping, status
+Boltzmann at 0.1.9 with githubci, ping, status
 ```
 
-You now have a complete hello world project in the `hello` directory, with commented source files to edit in place.
+As you can see, boltzmann informs you about what top-level dependencies it is adding
+and why those dependencies are present.
```

Comment:
(I note that the above output does not include any dependencies that are predicated on a feature being added, so "why those dependencies are present" isn't clear from the example)